crates/rspack_plugin_javascript/src/ast/stringify.rs

```diff
@@ -77,12 +77,10 @@ pub fn print(
       }
 
       let mut emitter = Emitter {
-        cfg: codegen::Config {
-          minify,
-          target,
-          ascii_only,
-          ..Default::default()
-        },
+        cfg: codegen::Config::default()
```

Comment:
This change is caused by https://github.com/swc-project/swc/commit/ee7575695de6dad140457ffb8bb8f0ac80c4dcdc#diff-7c02e040b4abd4d6a4c5c79c50c6976d065e7da0fe25a33f02b0da74a838cddb

Reply:
Looks good